app/controllers/import_controller.rb

```diff
@@ -297,6 +298,15 @@ def import_phone_field(value)
     e_164.blank? ? value : e_164
   end
 
+  def import_and_validate_int_phone_field(field, value, row_ind)
+    unless value == value&.gsub(/[^0-9.\-()+ ]/, '')
+      raise ValidationError.new("Value '#{value}' for '#{VALIDATION[field][:label]}' is not an acceptable value, please only use valid characters which \
+                                include digits, \".\", \"-\", \"(\", \")\", \"+\", \" \".", row_ind)
+    end
+
+    import_phone_field(value)
```

Comment:
I think that if we use `import_phone_field`, we may get some inconsistent results, since in that function, we parse the phone number as if it is a US number. For example:
```
+49 30 98321073
```
Is a German number. If the `+49` is included, it should parse fine, but if not, and the input is just `30 98321073`, then `Phonelib` will prepend a `+1` since it thinks the number is US based. I would hope that generally people will include the country code, but is that a totally safe bet? I feel like we are better off doing a non-US based parse.

Reply:
that's a valid concern, do you think it would make more sense to just not use `Phonelib` to parse it at all?

Reply:
Yeah if we never intend to call the number, it may make sense to just not parse, especially since in the UI we treat it as basically a free text field as long as you are using the desired characters.

Reply:
ok, PhoneLib is no longer used. I also switched to using model validation that way the validation behavior on API creates will be the same: https://github.com/SaraAlert/SaraAlert/pull/1069/commits/88ac0435a48b054947d19cbf9c2da2993c286ac9#diff-de292ba463e7050913b0b9c7fd4a0b30ad83439133001bf3a3121d7749046b88R4